server/routes/journeys/develop-an-initial-plan/next-review-date/tests.cy.ts

```diff
@@ -65,10 +66,11 @@ context('Make a Referral Journey', () => {
       .clear()
       .type(`${today.getDate()}/${today.getMonth() + 1}/${today.getFullYear()}`)
     cy.findByRole('button', { name: /choose date/i }).click()
-    const currentMonthLong = new Intl.DateTimeFormat('en-GB', { month: 'long' }).format(today)
-    cy.findAllByRole('button', { name: new RegExp(`Excluded date.*${currentMonthLong}.*`, 'i'), hidden: true }).should(
+    cy.findAllByRole('button', { name: /Excluded date.*/i, hidden: true }).should('have.length', today.getDate() - 1)
+    cy.findByRole('button', { name: /previous month/i }).click()
+    cy.findAllByRole('button', { name: /Excluded date.*/i, hidden: true }).should(
       'have.length',
-      1,
+      getDaysInMonth(new Date(today.getFullYear(), today.getMonth() - 1, 1)),
```

Comment:
It doesn't actually matter for this test, but getMonth actually returns the month by 0-index, so today.getMonth() - 1 is actually 2 months ago :)

Reply:
the date constructor also works on 0 indexes so -1 is needed to get the previous month

Reply:
Ah I had misread what you were doing 😅 sorry!